### PR TITLE
Update docs with env vars and docker-compose examples

### PR DESCRIPTION
**Pull request by alfredodeza**
Minor additions to the documentation, with docker-compose examples and environment variable descriptions.

On PyPI the package doesn't show any docs at all, wasn't sure if that was on purpose. A way to improve this is to read the contents of README.md into `long_description` in `setup.py`. I can follow up with that if that sounds OK.

### CODE REVIEW

**Comment by ryanpetrello** [comment]
> A way to improve this is to read the contents of README.md into long_description in setup.py

Please do :D

File: README.md [hunk 1]
```diff
@@ -45,8 +45,20 @@ to automatically discover open remote debugging sessions and connect to them:
 
 This will open a Python process that listens for new debugger sessions and
 automatically connects to them for you.  If your breakpoint is run on
-an _entirely different host_, you can optionally specify the hostname where
-`sbd-listen` is running:
+an _entirely different host_ (e.g. `10.0.0.1`) you can optionally specify the
+hostname where `sbd-listen` is running:
```
Comment (ryanpetrello) [inline README.md]: Typo `sdb`.

File: README.md [hunk 2]
```diff
@@ -56,6 +68,23 @@ an _entirely different host_, you can optionally specify the hostname where
 The `sbd-listen` tool also includes support for tab-completion and history
 tracking.
 
+Configuration with Environment Variables
+----------------------------------------
+
+It is possible to set ennvironment variables to configure `sdb`, which allows
```
Comment (ryanpetrello) [inline README.md]: typo `environment`

**Comment by ryanpetrello** [comment]
Fix up a few typos, then 👍 thanks!

File: README.md (hunk 1 above)
Comment (alfredodeza) [reply README.md]: I typo'ed the environment variables as well, unbelievable. You should publish an alias!

**Pull request merged**